Weather_Display_Matrix/openweather_graphics.py

```diff
@@ -0,0 +1,205 @@
+import time
+import displayio
+from adafruit_display_text.label import Label
+from adafruit_bitmap_font import bitmap_font
+
+TEMP_COLOR = 0xFFA800
+MAIN_COLOR = 0x9000FF  # weather condition
+DESCRIPTION_COLOR = 0x00D3FF
+CITY_COLOR = 0x9000FF
+HUMIDITY_COLOR = 0x0000AA
+WIND_COLOR = 0xCCCCCC
+
+cwd = ("/" + __file__).rsplit("/", 1)[
+    0
+]  # the current working directory (where this file is)
+
+small_font = cwd + "/fonts/Arial-12.bdf"
+medium_font = cwd + "/fonts/Arial-14.bdf"
+
+icon_spritesheet = cwd + "/weather-icons.bmp"
+icon_width = 16
+icon_height = 16
+scrolling_text_height = 24
+scroll_delay = 0.03
+
+
+class OpenWeather_Graphics(displayio.Group):
+    def __init__(
+            self,
+            display,
+            *,
+            am_pm=True,
+            units="imperial"
+    ):
+        super().__init__(max_size=3)
+        self.am_pm = am_pm
+        if units == "metric":
+            self.celsius = True
+            self.meters_speed = True
+        else:
+            self.celsius = False
+            self.meters_speed = False
```

Comment:
Would it make more sense to have units as a boolean like `metric_units=False` and then you could set:
```
self.celsius = metric_units
self.meters_speed = metric_units
```

Reply:
we had discussed yesterday how keeping it the current way is descriptive and allows for someone to add the default Kelvin mode if they want.